common/changes/office-ui-fabric-react/FocusAsync_2018-06-07-16-36.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Adding focus async to focus autofill when calling componentDidUpdate",
+      "type": "patch"
```

Comment:
Because the API surface has changed, this should be a minor

Reply:
done